ui/client/components/ValidationLabels.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import {v4 as uuid4} from "uuid";
+
+export default class ValidationLabels extends React.Component {
+
+  render() {
+    const {validators, value} = this.props
+    return validators.map(validator => validator.isValid(value) ? null :
+      <label key={uuid4()} className='validation-label'>{validator.message}</label>)
+  }
+}
```

Comment:
This component is to be used by node modal only, please move it to some specific subdirecotry in the hierarchy.

Reply:
Actually it can also be reused by other components, but for now I've moved it to modals subdirectory